src/figvalidationservice/figcrpords.yaml

```diff
@@ -439,5 +455,28 @@ definitions:
               type: string
       status_code:
         type: string
+      status_message:
+        type: string
+  dpsDocumentRequestBody:
+    type: object
+    required:
+    - server_name
+    - file_name
+    properties:
+      server_name:
+        type: string
+      file_name:
+        type: string
+  dpsDocumentResponse:
+    type: object
+    required:
+    - guid
+    - status_code
+    - status_message
+    properties:
+      guid:
+        type: string
+      status_code:
+        type: string
       status_message:
         type: string
```

Comment:
out of scope, but this file is missing a newline at the end. @CarolCarpenterJustice could you add one please :)